src/appkit.cs

```diff
@@ -22110,6 +22543,20 @@ partial interface NSGlyphInfo : NSCoding, NSCopying, NSSecureCoding {
 
 		[Export ("characterCollection")]
 		NSCharacterCollection CharacterCollection { get; }
+
+		[Mac (10,13)]
+		[Static]
+		[Export ("glyphInfoWithCGGlyph:forFont:baseString:")]
+		[return: NullAllowed]
+		NSGlyphInfo GlyphInfoWithCGGlyph (ushort glyph, NSFont font, string @string);
```

Comment:
That uses the new CGGlyph not the deprecated NSGlyph.

I'll fix the name. I'm open to a suggestion on how to note CGGlyph vs NSGlyph.  The problem is that we have using CGGlyph already, and some of those are really NSGlyph APIs.